app/javascript/dashboard/components/FormSection.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="flex flex-col items-start w-full gap-6">
+    <div class="flex flex-col w-full gap-4">
+      <h4 v-if="header" class="text-lg font-medium text-ash-900">
+        {{ header }}
+      </h4>
+      <div class="flex flex-row items-center justify-between">
+        <div class="flex-grow h-px bg-ash-200" />
+      </div>
+      <p v-if="description" class="mb-0 text-sm font-normal text-ash-900">
+        {{ description }}
+      </p>
+    </div>
+    <div v-if="$slots.default" class="flex flex-col w-full gap-6">
```

Comment:
```suggestion
    <div class="flex flex-col w-full gap-6">
```

We don't need this check, since this component should not be used without the default slot